gradle/libs.versions.toml

```diff
@@ -16,6 +16,7 @@ coroutines-js = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core-js", v
 coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "coroutines" }
 datetime = { module = "org.jetbrains.kotlinx:kotlinx-datetime", version = "0.3.2" }
 robolectric = { module = "org.robolectric:robolectric", version = "4.7.3" }
+material = { module = "com.google.android.material:material", version = "1.5.0" }
```

Comment:
nit: do you want to keep these alphabetically sorted?

Reply:
Good catch! Originally I meant for this dependency to be temporary, and forgot to fix this when I decided to leave it in for the sample.